lib/flight_job/version.rb

```diff
@@ -25,5 +25,5 @@
 # https://github.com/openflighthpc/flight-job
 #==============================================================================
 module FlightJob
-  VERSION = '1.1.0'
+  VERSION = '1.2.0'
```

Comment:
This needs updating.

Reply:
https://github.com/openflighthpc/flight-job/pull/3/commits/9e7aa3bc0fb68fb8ddfce896d8b10f03c04cc051